PixelDefinitions/params_dictionary.json

```diff
@@ -0,0 +1,29 @@
+// This file defines commonly used params - this does NOT mean they are added by default
+// In your pixel definitions, you can reference these params by their key
+// See TEMPLATE.json for an example
+{
+    "appVersion": {
+        "key": "appVersion",
+        "type": "string",
+        "description": "",
+        "pattern": "^[0-9]+\\.[0-9]+\\.[0-9]+(\\.[0-9]+)?$",
+        "examples": ["0.89.3"]
+    },
+    "atb": {
+        "key": "atb",
+        "type": "string",
+        "pattern": "v[0-9]{1,3}-[0-7]([a-z]{2})?",
+        "description": "Add to browser value, corresponding to the users install week and day"
+    },
+    "channel": {
+        "key": "channel",
+        "type": "string",
+        "description": "The channel the user is on",
+        "enum": ["dev", "canary", "beta", "preview", "stable", "unknown", "unpackaged"]
+    },
+    "remoteConfigEtag": {
+        "key": "remoteConfigEtag",
+        "description": "",
```

Comment:
Some minimal description here would help set a better precedent :) (and again, I know we have the same issue in other repos)